java/client-apis/pom.xml

```diff
@@ -18,7 +18,7 @@
     <parent>
         <artifactId>rocketmq-client-java-parent</artifactId>
         <groupId>org.apache.rocketmq</groupId>
-        <version>5.0.6-SNAPSHOT</version>
+        <version>5.0.7-SNAPSHOT</version>
```

Comment:
Could you set it to 5.0.6 ?

Example: https://github.com/apache/rocketmq-clients/blob/997d322e71a5f6d8bc189b88eea6caad211105f8/java/pom.xml#L29C5-L29C29